### DISCUSSION TITLE
alist新版3.28.0缺少配置

### DISCUSSION THREAD

**Discussion by 2928660938** [AlistGo/alist#5327] [Q&A] [answered]
alist新版3.28.0缺少配置因为安装D盘导致无法启动请求大佬瞄一眼一下
![Image_1696331596407](https://github.com/alist-org/alist/assets/81186890/0d7610f5-83a9-4398-a960-d6e424d3aa97)
![Uploading IMG_20231003_193154.jpg…]()

**Comment by xhofe** [comment, marked as answer]
把配置文件 `C:\Users\Administrator\AppData\Roaming\ci.nn.alist\config.json` 删了再启动。

**Reply by 2928660938** [reply]
好的，可以了，谢谢大哥，不好意思添麻烦了
